client/components/Buttons/CastButton.vue

```diff
@@ -22,9 +22,8 @@
             class="align-self-center active-button"
             :icon="!fab"
             :fab="fab"
-            small
-            disabled
```

Comment:
@ferferga This menu is non-functional, we shouldn't show it.

Reply:
@MrTimscampi https://matrix.to/#/!kPTHLknOmRaexLxUwT:matrix.org/$YnSGHQIObCdgeWwev5_5YoJ5ObJPIVRHmzUdlOEkfIs?via=matrix.org&via=t2bot.io&via=bonifacelabs.ca

This is the original reason for this, I don't care anyway honestly.